README.md

```diff
@@ -47,4 +47,4 @@ Implement the ability to edit a todo title on double click:
 
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_todo-app-with-api/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://SerhiiSemennikov.github.io/react_todo-app-with-api/) and add it to the PR description.
```

Comment:
Add this demo link for the PR description 
<img width="945" alt="image" src="https://github.com/user-attachments/assets/e2b65edf-98cd-41ae-b9a8-c70cb76d0f7a" />